Lib/imghdr.py

```diff
@@ -148,7 +148,7 @@ def testall(list, recursive, toplevel):
     import os
     for filename in list:
         if os.path.isdir(filename):
-            print(filename + '/:', end=' ')
+            print(filename + os.sep +':', end=' ')
```

Comment:
I don't know this kind of code churn is proper,
But if this is needed, what about this?

```python
print(f'{filename}{os.sep}:', end=' ')
```

Reply:
As a personal opinion, I don't see much improvement here in changing. It's just concatenating two separate variables and there are no other strings also as part of string like `a + " equals " + b` to `f"{a} equals {b}"` .